<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Combo criada na interface:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbo_endereco">
         <item>
          <property name="text">
           <string>Endereço 1</string>
          </property>
          <property name="icon">
           <iconset resource="arquivoRecursos.qrc">
            <normaloff>:/ico/ico/endereco1.png</normaloff>:/ico/ico/endereco1.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Endereço 2</string>
          </property>
          <property name="icon">
           <iconset resource="arquivoRecursos.qrc">
            <normaloff>:/ico/ico/endereco2.png</normaloff>:/ico/ico/endereco2.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Endereço 3</string>
          </property>
          <property name="icon">
           <iconset resource="arquivoRecursos.qrc">
            <normaloff>:/ico/ico/endereco3.png</normaloff>:/ico/ico/endereco3.png</iconset>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Combo criada via código:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbo_endCodigo"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="chk_item1">
         <property name="text">
          <string>Item 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chk_item2">
         <property name="text">
          <string>Item 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chk_item3">
         <property name="text">
          <string>Item 3</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QRadioButton" name="rad_credito">
         <property name="text">
          <string>Crédito</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rad_debito">
         <property name="text">
          <string>Débito</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rad_boleto">
         <property name="text">
          <string>Boleto</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_Confirmar">
         <property name="text">
          <string>Confirmar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>329</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="arquivoRecursos.qrc"/>
 </resources>
 <connections/>
</ui>
